core/domain/domain.en.yml

```diff
@@ -451,3 +451,76 @@ responses:
 
   utter_to_be_continued:
     - text: To be continued...
+
+  utter_daily_ci__acknowledge_fever:
+    - text: OK, I'm taking note that you have a fever today.
```

Comment:
Since we plan on removing dots at the end of messages, we should not add them in new messages.